microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -210,6 +210,11 @@ On desktop operating systems, Microsoft Edge is commonly identified by the `Edg`
 | Desktop (Windows/Mac/Linux) | `Edg` |
 | iPhone/iPad | `EdgiOS` |
 | Android (Mobile/tablet) | `EdgA`|
+| Legacy Edge* | `Edge`|

```

Comment:
I think the official name is "Microsoft Edge Legacy"
https://support.microsoft.com/en-us/microsoft-edge/what-is-microsoft-edge-legacy-3e779e55-4c55-08e6-ecc8-2333768c0fb0#:~:text=What%20is%20Microsoft%20Edge%20Legacy%3F%20Microsoft%20Edge%20Microsoft,Edge%20Legacy%20will%20end%20on%20March%209%2C%202021.

Reply:
fixing